test/root_component_test.dart

```diff
@@ -0,0 +1,75 @@
+@TestOn('browser')
```

Comment:
These tests are not comprehensive. Additional tests will be added in the next commit.